test/test-execute/exec-systemcallfilter-with-errno-number.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Test for SystemCallErrorNumber
+
+[Service]
+ExecStart=/bin/sh -x -c 'uname -a'
+Type=oneshot
+SystemCallFilter=~uname:255
```

Comment:
It would be better to have the tests check the return error codes, combining both other SystamCallErrorNUmber= cases, etc, and maybe try to cover regressions if they are not covered too!

Thank you @yuwata !

Reply:
OK. I will update.